app/views/admin/configs/show.html.erb

```diff
@@ -234,11 +224,59 @@
                                              placeholder: Constants::SiteConfig::DETAILS[:recaptcha_secret_key][:placeholder] %>
                           </div>
                         </div>
+                        <div class="mt-6">
+                          <% if SiteConfig.allow_both_email_signup_and_login %>
+                            <button
+                              class="crayons-btn crayons-btn--danger"
+                              data-action="click->config#activateEmailAuthModal"
+                              <%= SiteConfig.invite_only_mode ? 'disabled' : ''%>>
+                              <% if SiteConfig.invite_only_mode || authentication_enabled_providers.none? %>
+                                <span
+                                  class="crayons"
+                                  data-tooltip="<%= Constants::SiteConfig::DETAILS[:invite_only_mode][:cannot_disable_email] %>">
```

Comment:
Does this need to actually live _inside_ of SiteConfig constant? I think we can just inline it here. I'm suggesting this because I think it adds more complexity -- for example, I had to figure out what "this" mean in the "You cannot do this" text. Then I scrolled to this point in the file and figured out that the text was describing whether or not someone could click on the "disable" button, which was not clear to me. I'm also suggesting it because the rest of the SiteConfig values only include the `description` and `placeholder`, so adding a `cannot_disable_email` seems a bit odd, IMO. And because this is used by the `data-tooltip`, I think it it's okay to put it inline.
```suggestion
                                  data-tooltip="You cannot do this until you enable at least one other registration option.">
```